queue_and_stack/dll_stack.py

```diff
@@ -1,18 +1,26 @@
 import sys
-sys.path.append('../doubly_linked_list')
+
+sys.path.append("../doubly_linked_list")
 from doubly_linked_list import DoublyLinkedList
 
+
 class Stack:
     def __init__(self):
         self.size = 0
         # Why is our DLL a good choice to store our elements?
-        # self.storage = ?
+        self.storage = DoublyLinkedList(self.size)
```

Comment:
Also interested in how this still works with this argument of "self.size" being passed into here!